Framework/Kernel/inc/MantidKernel/Unit.h

```diff
@@ -662,6 +662,27 @@ class MANTID_KERNEL_DLL Temperature : public Empty {
   UnitLabel m_label;
 };
 
+//=================================================================================================
+/// Temperature in kelvin
```

Comment:
Comment needs updating
```suggestion
/// Atomic Distance in Angstroms
```